modules/ROOT/pages/upgrade.adoc

```diff
@@ -4,24 +4,28 @@ include::_attributes.adoc[]
 endif::[]
 
 == Before You Begin
+Because Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrades require a working installation, all environment prerequisites should already be met. As long as you (or your infrastructure team) successfully performed all prerequisite tasks listed in xref:prereq-hardware.adoc[Hardware Prerequisites], xref:prereq-software.adoc[Software Prerequisites], and xref:prereq-network.adoc[Network Prerequisites] prior to the installation of Anypoint Platform PCE, your environment can be upgraded successfully.
 
-Refer to the xref:release-notes/pce/anypoint-private-cloud-2.0.3-release-notes#upgrade-paths[Upgrade Paths) section in the Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) release notes for information about upgrading Anypoint Platform PCE to the most recent version.
+Refer to the xref:release-notes::pce/anypoint-private-cloud-2.0.3-release-notes#upgrade-paths[Upgrade Paths) section in the Anypoint Platform PCE release notes for information about upgrading Anypoint Platform PCE to the most recent version.
 
 See xref:1.6@upgrade.adoc[About Migrating Anypoint Platform Private Cloud] for information on upgrading to version 1.6.1. See xref:1.7@upgrade.adoc[About Migrating Anypoint Platform Private Cloud] for information on upgrading to version 1.7.1.
 
-[WARNING]
-====
-Before upgrading:
+If you are using AWS, obtain the keys used to run the AWS provisioner. These are required for access EC2, VPC, and S3.
 
-* Ensure your environment meets all requirements listed in the xref:install-checklist[Product Prerequisites].
-* Perform a backup of your system as described in xref:backup-and-disaster-recovery.adoc[Configure Backup and Restore]. If you are upgrading from an older version that requires multiple intermediate upgrades you must backup each intermediate version before proceeding to the next.
+Contact MuleSoft professional services prior to upgrading for assistance.
 
-During the upgrade process, there will be some downtime as each node is updated. 
-====
+== Procedure
 
-The following steps describe how to migrate Anypoint Platform PCE and the Anypoint Platform PCE infrastructure. You must also upgrade your applications to work with the new version of Anypoint Platform PCE. Contact your MuleSoft customer success representative for more information.
+The following steps describe how to upgrade Anypoint Platform PCE and the Anypoint Platform PCE infrastructure. 
 
-== Procedure
+[NOTE]
+====
+* During an upgrade, all Anypoint Platform PCE components are updated.
+* The upgrade process causes downtime for each node as it is updated.
+* The upgrade process does not include your applications and data. You must upgrade your applications to work with the new version of Anypoint Platform PCE.
```

Comment:
Not sure what you are trying to say here. Runtime applications are independant from PCE upgrades.

Reply:
I removed this line - thank you!